### PR TITLE
Add Administrative info, personal pronoun

### LINKED ISSUE #369: Add dietary restrictions, shirt size to single hacker view
Single hacker view should contain the following pieces of information when an admin views it.

### PR DESCRIPTION
**Pull request by pierreTklein**
# Changes

- Fixes #369 
- Add padding on bottom of single hacker page
- Add personal pronoun of user
- Add age, shirt-size, personal pronoun, and a link to call the user via their phone number.

# Screenshot
localhost_1337_application_view_5c0dd463d95414ef5efd14cd

### CODE REVIEW

File: src/SingleHacker/SingleHackerLink.tsx [hunk 1]
```diff
@@ -3,17 +3,24 @@ import * as React from 'react';
...
+  label,
+  newTab,
+}) => {
   return (
     <Box width={[1, 1 / 2]}>
       <strong>{label}</strong>:{' '}
       {link ? (
-        <a href={link} target="_blank">
-          {link}
+        <a href={link} target={newTab ? '_blank' : ''}>
```
Comment (erickzhao) [inline src/SingleHacker/SingleHackerLink.tsx]: Do we need this level of granularity for the `newTab` prop? I would rather just assume that we want to open anything that's a URL in a new tab, and others (e.g.`mailto:`, `tel:`) in the same one.

Comment (pierreTklein) [reply src/SingleHacker/SingleHackerLink.tsx]: Do we use a regexp for http:// and https:// ?

Comment (erickzhao) [reply src/SingleHacker/SingleHackerLink.tsx]: We can try to leverage this API? https://developer.mozilla.org/en-US/docs/Web/API/URL

We can check if it constructs and if its protocol is `http://` or `https://`

image

Comment (pierreTklein) [reply src/SingleHacker/SingleHackerLink.tsx]: Done

**Pull request merged**